src/main/java/org/assertj/vavr/internal/Maps.java

```diff
@@ -124,6 +127,51 @@ public static Maps instance() {
         }
     }
 
+    /**
+     * Verifies that the actual map contains only the given entries and nothing else, <b>in order</b>.<br>
+     * This assertion should only be used with map that have a consistent iteration order (i.e. don't use it with
+     * {@link io.vavr.collection.HashMap}).
+     *
+     * @param <K>     key type
+     * @param <V>     value type
+     * @param info    contains information about the assertion.
+     * @param actual  the given {@code Map}.
+     * @param entries the given entries.
+     * @throws NullPointerException     if the given entries array is {@code null}.
+     * @throws AssertionError           if the actual map is {@code null}.
+     * @throws IllegalArgumentException if the given entries array is empty.
+     * @throws AssertionError           if the actual map does not contain the given entries with same order, i.e. the actual map
+     *                                  contains some or none of the given entries, or the actual map contains more entries than the given ones
+     *                                  or entries are the same but the order is not.
+     */
+    public <K, V> void assertContainsExactly(AssertionInfo info, Map<K, V> actual,
+                                             @SuppressWarnings("unchecked") Tuple2<? extends K, ? extends V>... entries) {
+        doCommonContainsCheck(info, actual, entries);
+        if (actual.isEmpty() && entries.length == 0) return;
+        failIfEmpty(entries);
+        assertHasSameSizeAs(info, actual, entries);
+
+        final Map<K, V> expectedEntries = LinkedHashMap.ofEntries(entries);
+        final Map<K, V> notExpected = actual.filter(entry -> !expectedEntries.contains(entry));
+        final Map<K, V> notFound = expectedEntries.filter(entry -> !actual.contains(entry));
+
+        if (notExpected.isEmpty() && notFound.isEmpty()) {
+            // check entries order
+            int index = 0;
+            for (K keyFromActual : actual.keySet()) {
+                if (!areEqual(keyFromActual, entries[index]._1)) {
```

Comment:
Maybe add `areNotEqual` method? It makes reading easier.

Reply:
Good point